tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -117,3 +117,6 @@ cc-tdx-td-shim-tarball:
 
 cc-tdx-tdvf-tarball:
 	${MAKE} $@-build
+
+cc-sev-ovmf-tarball:
+	${MAKE} $@-build
```

Comment:
@Alex-Carter01, could you add this as part of the cc-tarball target?

Reply:
Ah, one important thing, by the moment we add this to the tarball, we should also handle cleaning up whatever was added as part of this: https://github.com/kata-containers/kata-containers/blob/b9c0f7fb09a9f89fe370be2382b8cc911ba4a9b7/tools/packaging/kata-deploy-cc/scripts/kata-deploy.sh#L186-L201

Please, double-check that everything you generate is cleaned up there and, if not, add there what has to be cleaned up.